src/etc/pre-commit.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/env bash
```

Comment:
Hmm, I normally see this as `/usr/bin/env`. Do you know if there's any difference between the two? I just checked and they're both present on my machine so there might not be.

Reply:
You probably (definitely) have the right thing, i just edited the shebang without thinking